docs/rules/arrow-parens.md

```diff
@@ -87,7 +87,7 @@ a.then((foo) => { if (true) {}; });
 
 #### If Statements
 
-One benefits of this option is that it prevents the incorrect use of arrow functions in conditionals:
+One of benefits of this option is that it prevents the incorrect use of arrow functions in conditionals:
```

Comment:
Already fixed this typo in a subsequent batch of updates, but currently struggling with current PR process so if possible please allow it to slip.